src/test/cypress/package.json

```diff
@@ -26,8 +26,8 @@
     "scripts": {
         "cypress:open": "cypress open",
         "cypress:run": "cypress run --browser=chrome",
-        "cypress:record:mysql": "npx cypress-cloud run --record --ci-build-id \"${SORRY_CYPRESS_BRANCH_NAME} #${SORRY_CYPRESS_BUILD_ID} (MySQL)\"",
-        "cypress:record:postgres": "npx cypress-cloud run --record --ci-build-id \"${SORRY_CYPRESS_BRANCH_NAME} #${SORRY_CYPRESS_BUILD_ID} (Postgres)\"",
+        "cypress:record:mysql": "npx cypress-cloud run --record --ci-build-id \"${SORRY_CYPRESS_BRANCH_NAME} #${SORRY_CYPRESS_BUILD_ID} ${SORRY_CYPRESS_RERUN_COUNT} (MySQL)\"",
```

Comment:
Where does the `SORRY_CYPRESS_RERUN_COUNT` come from now?

Reply:
It comes from here: https://github.com/ls1intum/Artemis/blob/82523a8d192cd4ad89d4b2503abbec78e49a464d/docker/cypress.yml#L27

It's a bamboo variable, which hold the count of reruns (on the first rerun it's null, but then a number). This will allow to rerun the failed builds in the future, since currently we use the same build id for a rerun which causes the run to fail.